indico/modules/events/controllers/base.py

```diff
@@ -25,21 +25,31 @@ def _process_args(self):
             raise NotFound(_('This event has been deleted.'))
 
 
-class RHDisplayEventBase(RHEventBase):
+class RHProtectedEventBase(RHEventBase):
+    """Base class for events that check if he user can access it.
+
+    This includes unauthenticated users who have access to the event
+    for any other reason (e.g. due to a whitelisted IP address).
+    """
+    def _check_access(self):
```

Comment:
add an empty line before this one (see [PEP257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings))